coral/docs/mock-api-for-development.md

```diff
@@ -0,0 +1,53 @@
+# Mocking an API for development
+
+We're using [MSW](https://mswjs.io/) to mock our API in tests. We **also** have setup a [service-worker](src/services/api-mocks/browser.ts). This can be used to mock API calls in the component during development.
+
+We recommend working with a [remote API](development-with-remote-api.md). But there may be reasons you want to work on Coral without setting up one. It's a bit of extra work. **Also, please take care not to commit these mocks 🙏.**
+
+## How to mock an API
+
+In this example, we assume you want to mock the [`team-api.ts`](../src/domain/team/team-api.ts).
```

Comment:
TIL writing relative paths links in an `md` file in a github project will make those paths absolute in the remote repo :o Nice!